test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,30 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+//I believe we are writing to servoPin, which I set to be PA_1
+CAN can1(PA_0, servoPin);
+float pulseWidthToSend;
```

Comment:
Prefer not having floating global variables in the class implementation .cpp file. You don't need a CAN instance inside the TutorialServo class and you can make `pulseWidthToSend` a local variable inside the function you intend to use it in.

Reply:
Yep, realized I don't need them at all. Fixed in 0cba407156933737ef1cbe0b85f325a7b83d9f0a